client/src/List.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import gameList from './gameList.json'
+
+export default function List() {
+  return (
+    <ContentWrapper>
+      <h2>My library</h2>
+      <ul>
+        {gameList.map((gameList) => (
+          <li>{gameList.title}</li>
+        ))}
+      </ul>
+    </ContentWrapper>
+  )
+}
+
+const ContentWrapper = styled.main`
```

Comment:
there's also some css in the globalstyles for the `<main>`, in case you want to have it all in one place?